src/config/constants/tokens.ts

```diff
@@ -1,1959 +1,1815 @@
+    'HTB',
+    'Hotbit Token',
+    'https://www.hotbit.io/',
+  ),
+  woo: new Token(
+    MAINNET,
+    '0x4691937a7508860F876c9c0a2a617E7d9E945D4B',
+    18,
+    'WOO',
+    'Wootrade Network Token',
+    'https://woo.network/',
+  ),
+  $dg: new Token(
+    MAINNET,
+    '0x9Fdc3ae5c814b79dcA2556564047C5e7e5449C19',
+    18,
+    '$DG',
+    'Decentral Games Token',
+    'https://decentral.games/',
+  ),
+  safemoon: new Token(
+    MAINNET,
+    '0x8076C74C5e3F5852037F31Ff0093Eeb8c8ADd8D3',
+    9,
+    'SAFEMOON',
+    'Safemoon Token',
+    'https://safemoon.net/',
+  ),
+  axs: new Token(
+    MAINNET,
+    '0x715D400F88C167884bbCc41C5FeA407ed4D2f8A0',
+    18,
+    'AXS',
+    'Binance-Pegged Axie Infinity Shard',
+    'https://axieinfinity.com/',
+  ),
+  c98: new Token(
+    MAINNET,
+    '0xaec945e04baf28b135fa7c640f624f8d90f1c3a6',
+    18,
+    'c98',
+    'Coin98 Token',
+    'https://coin98.com/',
+  ),
+  pots: new Token(
+    MAINNET,
+    '0x3Fcca8648651E5b974DD6d3e50F61567779772A8',
+    18,
+    'POTS',
+    'Moonpot Token',
+    'https://moonpot.com/',
+  ),
+  gnt: new Token(
+    MAINNET,
+    '0xf750a26eb0acf95556e8529e72ed530f3b60f348',
+    18,
+    'GNT',
+    'GreenTrust Token',
+    'https://www.greentrusttoken.com/',
+  ),
+  rusd: new Token(
+    MAINNET,
+    '0x07663837218a003e66310a01596af4bf4e44623d',
+    18,
+    'rUSD',
+    'rUSD Token',
+    'https://appv2.rampdefi.com/#/',
+  ),
+  bp: new Token(
+    MAINNET,
+    '0xacb8f52dc63bb752a51186d1c55868adbffee9c1',
+    18,
+    'BP',
+    'BunnyPark Token',
+    'https://www.bunnypark.com/',
+  ),
+  sfund: new Token(
+    MAINNET,
+    '0x477bc8d23c634c154061869478bce96be6045d12',
+    18,
+    'SFUND',
+    'Seedify Fund Token',
+    'https://seedify.fund/',
+  ),
+  naos: new Token(
+    MAINNET,
+    '0x758d08864fb6cce3062667225ca10b8f00496cc2',
+    18,
+    'NAOS',
+    'NAOSToken',
+    'https://naos.finance/',
+  ),
+}
+
+export const testnetTokens = {
+  wbnb: new Token(
+    TESTNET,
+    '0xae13d989dac2f0debff460ac112a837c89baa7cd',
+    18,
+    'wBNB',
+    'Wrapped BNB',
+    'https://www.binance.com/',
+  ),
+  cake: new Token(
+    TESTNET,
+    '0xa35062141Fa33BCA92Ce69FeD37D0E8908868AAe',
+    18,
+    'CAKE',
+    'PancakeSwap Token',
+    'https://pancakeswap.finance/',
+  ),
+  busd: new Token(
+    TESTNET,
     '0xeD24FC36d5Ee211Ea25A80239Fb8C4Cfd80f12Ee',
     18,
     'BUSD',
     'Binance USD',
+    'https://www.paxos.com/busd/',
   ),
+  syrup: new Token(
+    TESTNET,
+    '0xfE1e507CeB712BDe086f3579d2c03248b2dB77f9',
+    18,
+    'SYRUP',
+    'SyrupBar Token',
+    'https://pancakeswap.finance/',
+  ),
+  bake: new Token(
+    TESTNET,
+    '0xE02dF9e3e622DeBdD69fb838bB799E3F168902c5',
+    18,
+    'BAKE',
+    'Bakeryswap Token',
+    'https://www.bakeryswap.org/',
+  ),
+  // Not sure why - in our original tokens.ts, all of the below have testnet entries but the address points to the bakeryswap token
+  usdt: new Token(
```

Comment:
Not sure why - in our original `tokens.ts`, all of the testnet tokens below have addresses, but they are pointing to the bakeryswap token. Should I just remove these entries?